pinot-query-planner/src/main/java/org/apache/calcite/rel/rules/PinotWindowExchangeNodeInsertRule.java

```diff
@@ -171,4 +195,55 @@ private boolean isPartitionByOnlyQuery(Window.Group windowGroup) {
     }
     return isPartitionByOnly;
   }
+
+  /**
```

Comment:
Can you leave a TODO for support for column-less tuples?

Reply:
@ankitsultana can you elaborate a bit on what kind of queries you expect will need column-less tuples?

Reply:
Oh I was alluding to our discussion on Slack. Later on we might wanna support column less records in the intermediate stages in which case we may not need to customize the window/project transpose rule and we may be able to support empty project operators.

Reply:
got it, added a TODO